pkg/util/querylimits/middleware.go

```diff
@@ -27,11 +27,8 @@ func (l *queryLimitsMiddleware) Wrap(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		limits, err := ExtractQueryLimitsHTTP(r)
 		if err != nil {
-			w.WriteHeader(http.StatusBadRequest)
-			if _, err := w.Write([]byte(err.Error())); err != nil {
-				level.Error(util_log.Logger).Log("msg", "error in queryLimitsMiddleware Wrap", "err", err)
-			}
-			return
```

Comment:
Is there a nice way we could still notify users that the query limits contents were broken?

Reply:
I think cloud insights with some alerting.